core/src/main/java/io/kestra/core/models/property/Property.java

```diff
@@ -37,6 +37,7 @@ public class Property<T> {
         .copy()
         .configure(SerializationFeature.WRITE_DURATIONS_AS_TIMESTAMPS, false);
 
+    @Getter
```

Comment:
Please no, this is by design to force dev to render the property, we discuss about it this is a strong opinionated move